content/work/shopify.md

```diff
@@ -37,19 +37,18 @@ summary: |
 {% import 'layout.macros.njk' as layout %}
 {% import 'quotes.macros.njk' as quotes %}
 
+
 {{ layout.title('What We Did') }}
 
 {% call embed.media_block(
-  media=embed.screen(
-    notch='0',
-    content=embed.img(
-      src='work/shopify/shopify-phone.jpg',
-      alt='Miriam speaking on video for a virtual training session',
-      width=744 ,
-      height=1611,
-      sizes='media'
-    )
-  )
+  media=embed.img(
+    src='work/shopify/shopify.jpg',
+    alt='Miriam speaking in the UX Summer Shopify Series',
+    attrs={
+      'width':'1800',
+      'height':'807'
```

Comment:
I didn't think `width` and `height` were needed anymore. @mirisuzanne?

Reply:
I think you're right, the new responsive-image tool doesn't require these?

Reply:
removed these.